docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -690,6 +690,7 @@ Possible values are:
 | `mainnet` | Eth 2.0 | Production  | Intended main network that targets future mainnet configuration.    |
 | `medalla` | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
 | `minimal` | Eth 2.0 | Test        | Used for local testing and development networks.                    |
+| `spadina` | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
```

Comment:
Spadina isn't meant to be a long-term testnet, wonder if we should note this somehow:
```suggestion
| `spadina` | Eth 2.0 | Test        | Ephemeral multi-client testnet maintained by the Goerli testnet initiative.   |
```